packages/next/src/lib/metadata/generate/opengraph.tsx

```diff
@@ -260,16 +260,16 @@ export function TwitterMetadata({
       <Meta name="twitter:title" content={twitter.title?.absolute} />
       <Meta name="twitter:description" content={twitter.description} />
       {twitter.images
-        ? twitter.images.map((image) => (
-            <>
+        ? twitter.images.map((image, index) => (
+            <React.Fragment key={index}>
```

Comment:
Maybe use `image.url + image.alt` as the key? These shouldn't be keyed off the index.

cc @huozhi

Reply:
My bad, is it ok to use `!` here?
```tsx
{twitter.images
        ? twitter.images.map((image) => (
            <React.Fragment key={image.url! + image.alt!}>
              <Meta name="twitter:image" content={image.url} />
              <Meta name="twitter:image:alt" content={image.alt} />
            </React.Fragment>
          ))
        : null}
```

Reply:
Should be ok to use `index` since `<head />` doesn't change while application is running. I may be wrong but it's also used here: https://github.com/vercel/next.js/blob/canary/packages/next/src/lib/metadata/generate/icons.tsx

So maybe something like this instead?:
```suggestion
            <React.Fragment key={`image-${index}`}>
```

https://reactjs.org/docs/lists-and-keys.html :
> We don’t recommend using indexes for keys if the order of items may change.

Reply:
We can use index as key here since the entire metadata component will re-mount for each request, there's a key prop on the toplevel so here is fine. Basically just need to surpass the warning